googleapis/google-cloud-python
# BigQuery: Add StandardSqlDataTypes enum to BigQuery

Closes #8495.

This PR adds a convenience enum with scalar SQL data types (as per issue description).

### How to test
The issue is self-explanatory. Just make sure that the enum contains all the right fields that we want in it.

**Update:** Additionally, make sure that the docs for the new enum are in order.
